documentation/source/quickstart.rst

```diff
@@ -23,12 +23,6 @@ Installation via PIP
 
 Cocotb can be installed by running
 
-.. code-block:: bash
-
-    pip3 install cocotb
```

Comment:
I'm not so sure about this one - the fact that we don't support Python 2 anymore does not mean that the user cannot have both versions installed still and needs to get the Python-3-pip via ``pip3``.

Reply:
Perhaps I should remove the other leg then to remove any confusion?